Formula/lib/libpipewire.rb

```diff
@@ -0,0 +1,64 @@
+class Libpipewire < Formula
+  desc "Server and user space API to deal with multimedia pipelines"
+  homepage "https://pipewire.org"
+  url "https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.5.83/pipewire-1.5.83.tar.gz"
```

Comment:
This is an unstable version (release candidate for 1.6) - https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/1.5.83

Should use 1.4.9 and livecheck should also use even minor versions.